🌐 Update German translation for `docs/de/docs/fastapi-people.md`

nilslindemann opened the pull request:
Based on #3048.

← `reference/testclient.md` (#10843)
→ `resources/index.md` (#10497)
→ `project-generation.md` (#10851)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

Commit:
German translation of docs/fastapi-people.md

* Full translation
* Based on #3048
* In sync with current en/docs/fastapi-people.md
* Hand-translated (Google translate rarely used)
* Checked for errors using LanguageTool
* Manually checked

tiangolo commented:
📝 Docs preview for commit e04482df9abbad938ddcbd87f07af65129d37160 at: https://0abd39ab.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

tiangolo commented:
📝 Docs preview for commit e0d2cb27014dad126fbd87300651397c6a6727de at: https://09b37800.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

tiangolo commented:
📝 Docs preview for commit 8c66720b51f8a45b3736dee941910219d661b6ae at: https://86f60efb.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

tiangolo commented:
📝 Docs preview for commit ae15753e5e2771185d6447866e179be560afe484 at: https://07cdcc56.fastapitiangolo.pages.dev

Commit:
Translate links pointing to help-fastapi.md

That document is now translated, see https://github.com/tiangolo/fastapi/pull/10455

tiangolo commented:
📝 Docs preview for commit 2501e2b88596449a5a07d1e92ac4c59c7b773e55 at: https://9e972431.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

tiangolo commented:
📝 Docs preview for commit bb42b20ea353673767bd9666bd0f1e4a32bf31f6 at: https://dfa1a6b4.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

tiangolo commented:
📝 Docs preview for commit 96ec54096973e4353d7b11a878fdda6955004fae at: https://9db488ff.fastapitiangolo.pages.dev

Commit:
Sync with #10392

tiangolo commented:
📝 Docs preview for commit 0974a4eee88673066530a0279aea59923a083a28 at: https://45b32a7d.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

tiangolo commented:
📝 Docs preview for commit cc924dd6cb0983c5742a1b56534819f0f0741fd5 at: https://c7ae8cbc.fastapitiangolo.pages.dev

Commit:
Translate links pointing to `contributing.md`

That document is now translated, see #10487

tiangolo commented:
📝 Docs preview for commit 2a95573676f4fe7269f462150521715930d509df at: https://c55c9a34.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

tiangolo commented:
📝 Docs preview for commit 41b63566bf6e55d414657224157d47ecc198e656 at: https://d4ff54d5.fastapitiangolo.pages.dev

Commit:
Use typographic quotes

tiangolo commented:
📝 Docs preview for commit 6663a3862752e9782f293bc330e7c678652d909e at: https://6ce95d5c.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

tiangolo commented:
📝 Docs preview for commit 6021189ee921e68cb10ae0b07e474e6a876fc3ec at: https://fd4f2439.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

tiangolo commented:
📝 Docs preview for commit d1de6502da38da262d898809307979326afe4326 at: https://cbaded84.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

Commit:
Merge branch 'tiangolo:master' into de/docs/fastapi-people.md

tiangolo commented:
📝 Docs preview for commit 1711b2db583b5f61c9656c5b97c5295be45dbb18 at: https://3a319c87.fastapitiangolo.pages.dev

tiangolo commented:
Awesome @nilslindemann! :rocket: 

As you are the champion of German translations and you have done so many, and with so much attention to detail, I'm gonna go ahead and merge this. :tada: 

If others see things that could be improved later, we can check that in new PRs. :nerd_face:
